tpl/template_funcs.go

```diff
@@ -1132,6 +1134,35 @@ func highlight(in interface{}, lang, opts string) (template.HTML, error) {
 	return template.HTML(helpers.Highlight(html.UnescapeString(str), lang, opts)), nil
 }
 
+// execute a command and returns its output as string
+func exec(in interface{}, args ...string) (string, error) {
+	str, err := cast.ToStringE(in)
+
+	if err != nil {
+		return "", err
+	}
+
+	// Check if command is accepted (in white list)
+	commandAccepted := false
+	for _, validCommand := range viper.GetStringSlice("ExecWhiteList") {
+		if str == validCommand {
+			commandAccepted = true
+		}
+	}
+
+	if !commandAccepted {
+		jww.ERROR.Printf("Executing %s is not allowed. Check execWhiteList settings.", str)
+		return "", nil
```

Comment:
>> +
>> +    if !commandAccepted {
>> +        jww.ERROR.Printf("Executing %s is not allowed. Check
> execWhiteList settings.", str)
>> +        return "", nil
> 
> Return an error instead of logging it.

Done. The test for the failure case looks a bit awkward now, let me know 
if it can be improved (never did anything with Go before).